src/main/java/ejercicio/banco/service/AccountServiceImpl.java

```diff
@@ -34,13 +40,20 @@ public Account findAccount(int accountId) {
         } catch (AccountNotFoundException e) {
             e.printStackTrace();
             account = null;
+        } catch (SQLException e) {
+            e.printStackTrace();
         }
         return account;
     }
 
     // Find accounts by string
-    public List<Account> findAccounts(String filename, String name) {
-        return repository.findByName(filename, name);
+    public List<Account> findAccounts(String filename, String name){
+        try {
```

Comment:
This is the try-catch that must be moved to the repository.

Reply:
Done on last commit, I think